hacking/build_library/build_ansible/command_plugins/plugin_formatter.py

```diff
@@ -183,9 +183,9 @@ def get_plugin_info(module_dir, limit_to=None, verbose=False):
         if module_path.endswith('__init__.py'):
             continue
 
-        # Do not list blacklisted modules
+        # Do not list modules on the reject list
         module = os.path.splitext(os.path.basename(module_path))[0]
-        if module in plugin_docs.BLACKLIST['MODULE'] or module == 'base':
+        if module in plugin_docs.REJECTLIST['MODULE'] or module == 'base':
             continue
 
         # If requested, limit module documentation building only to passed-in
```

Comment:
Please do not merge this as plugin_formatter is being removed.  Instead let me know about such things and I'll make sure to add them into the new code base if that code still exists.

Reply:
if you remove i'll rebase